Resources/Prototypes/Entities/Structures/Storage/Canisters/gas_canisters.yml

```
@@ -789,6 +789,60 @@
   - type: Lock
     locked: true
 
+- type: entity
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 0

(indentation)